docs/modules/ROOT/pages/references/service-maturity.adoc

```diff
@@ -0,0 +1,142 @@
+
+== Iteration: Cluster Version Upgrade
+
+As an *End-User* I can:
+
+* schedule an automated upgrade of my Cloud Kubernetes environment to the next minor version.
+
+As a *Technical VSHNeer* I can:
+
+* trigger an automated upgrade of the Cloud Kubernetes environment to the next minor version.
+
+== Iteration: Cluster Backup and Restore
+
+As an *End-User* I can:
+
+* rely on automated backups of my Cloud Kubernetes environment configuration and cluster state.
+* schedule backups cluster state backups to run automatically, at a time and frequency that suits my needs.
+* configure data retention policies to ensure that backups are kept for the desired length of time.
+* quickly and easily restore my Cloud Kubernetes environment to a previous state, including all cluster configuration.
+* test restores in a test environment before performing a full restore, to ensure that the backup is fully restorable.
+
+== Iteration: Cluster-Level Metrics and Alerting
+
+As an *End-User* I can:
+
+* access a metrics dashboard to get an overview of the Cloud Kubernetes environment and health.
+
+As a *Technical VSHNeer* I can:
+
+* have a metrics service that will help me keep track of the health and performance of the Cloud Kubernetes environment.
+* enable alerting on pre-defined SLIs to VSHN so that VSHN can resolve incidents with priority.
+* see SLO reports so that I can get an overview of how well my Cloud Kubernetes environment performs.
+* access operations runbooks so that I can resolve upcoming alerts.
+* access a metrics dashboard to get an overview of the Cloud Kubernetes environment and health.
+
+== Iteration: End-User Metrics and Alerting
+
+As an *End-User* I can:
+
+* enable a service to automatically collect and aggregate metrics from my Cloud Kubernetes environment and applications, providing a unified view of all metrics.
+* have metrics visualized in real-time, with interactive charts and graphs, providing insights into the behavior and performance of my Cloud Kubernetes environment and the applications.
+* configure alarms to alert me when specific conditions are met, such as when an application experiences an error.
+* create custom dashboards to view metrics that are most important to me, and to quickly identify areas of concern.
+
+== Iteration: Cluster Autoscaling
+
+As an *End-User* I can:
+
+* enable autoscaling of the cluster so that new worker nodes get added when needed and removed again when not needed anymore.
+* define a lower and upper bound of the amount of worker nodes.
+* specify what kind of worker nodes are part of the autoscaling.
+
+== Iteration: Logs
+
+As an *End-User* I can:
+
+* enable collection of historic logs of my applications running on the cluster.
+* access historic logs of my application running on the cluster.
+* configure retention time of historic logs.
+
+As a *Technical VSHNeer* I can:
+
+* access the logs of the Kubernetes control plane via a user-friendly interface.
```

Comment:
Nice _user-friendly_ is only relevant for VSHNeers but is not mentioned for end-users 🤣

Reply:
That wasn't the intention, and slipped through my fingers. I'd remove "user-friendly" here, as this is not a well-defined wording. What does "user-friendly" actually mean? Define "user-friendly"...

Reply:
Maybe instead of user-friendly interface we can write "access the logs of the Kubernetes control plane via a UI dashboard" or "via an intereactive dashboard". What do you think about that? 

And on that note, we wrote it a bit differently for the End-User ("access historic logs...") - maybe we can also edit this sentence for the End-User? i.e to add at the end of this sentence the "via a UI dashboard" or "via an intereactive dashboard".

Another thing I think we should consider: do we want to have a feature for the End-User application mentioned here as well or if we decide it will be a standalone product, then maybe we should remove it from this iteration?